core-plugins/docs/Deduplicate-batchaggregator.md

```diff
@@ -7,30 +7,39 @@ De-duplicates records either using one or more fields or by using the record as
 choosing a record out of the duplicate records based on a filter field. Supported logical functions are `any`, `max` and `min`.
 `min` and `max` must be operated only on numeric filter fields.
 
-The `first` and `last` are deprecated and will be removed in later versions of the platform.
-
 The `any` logical function will match any of the records where the value for the specified field is non-null.
-If not found, a record where the value of this field is null will be returned.
+If not found, a record where the value of this field is null will be returned. 
+
+The `first` and `last` are deprecated and will be removed in the future versions of the platform.
```

Comment:
should we say first and last logical functions? (sorry if I missed this last time...) Either Delete "The" or add logical functions.

Reply:
yeah that makes sense. I'll add logical functions